pkg/config/config.go

```diff
@@ -274,6 +274,9 @@ type LibpodConfig struct {
 	// files.
 	StaticDir string `toml:"static_dir"`
 
+	// StopTimeout amount of time to wait for container processes to exit before sending kill signal
+	StopTimeout uint `toml:"stop_timeout"`
```

Comment:
could it be a float?

Reply:
Inside of podman right now it is an uint.  I don't think we need that fine grain of control.